metron-platform/metron-parsing/metron-parsers-common/src/main/java/org/apache/metron/parsers/utils/DateUtils.java

```diff
@@ -36,6 +36,14 @@
  */
 public class DateUtils {
 
```

Comment:
We mention specs here, we should link to them

Reply:
Done